### PR TITLE
[OSDOCS-8979]: Update etcd recovery procedure

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.14+

Issue: https://issues.redhat.com/browse/OSDOCS-8979

Link to docs preview: https://79311--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state (Scroll to the end of step 23)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/dr-restoring-cluster-state.adoc [hunk 1]
```diff
@@ -760,7 +760,7 @@ etcd-ip-10-0-154-194.ec2.internal                2/2     Running     0
 etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0          9h
 ----
 
-To ensure that all workloads return to normal operation following a recovery procedure, restart each pod that stores `kube-apiserver` information. This includes {product-title} components such as routers, Operators, and third-party components.
+To ensure that all workloads return to normal operation following a recovery procedure, restart all control plane nodes and worker nodes.
```
Comment (geliu2016) [inline modules/dr-restoring-cluster-state.adoc:763]: As my understanding, worker nodes need not be restarted, is it make sense?

Comment (lahinson) [reply modules/dr-restoring-cluster-state.adoc]: @geliu2016 I updated line 763 to state that only control plane nodes need to be restarted. Thanks!

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**